saleor/core/telemetry/saleor_attributes.py

```diff
@@ -0,0 +1,25 @@
+from typing import Final
+
+SALEOR_ENVIRONMENT_DOMAIN: Final = "saleor.environment.domain"
+
+OPERATION_NAME: Final = "operation.name"
+COMPONENT: Final = "component"
+SPAN_TYPE: Final = "span.type"
+
+# GraphQL
+GRAPHQL_DOCUMENT_FINGERPRINT: Final = "graphql.document_fingerprint"
+GRAPHQL_OPERATION_IDENTIFIER: Final = "graphql.operation.identifier"
```

Comment:
OTEL's semantic conventions use [graphql.operation.name](https://opentelemetry.io/docs/specs/semconv/attributes-registry/graphql/#graphql-operation-name) to identify the name of the executed operation. Should we keep our "identifier" or switch to OTEL's version? We could also keep it for now in this PR, but as I'm adding attrs for new metrics, I was thinking about going the OTEL way in my [PR](https://github.com/saleor/saleor/pull/17616/files) and switch to `graphql.operation.name` for new metrics.

Reply:
I just saw that you already use both in the view. So we can discuss the metrics attrs in the other PR as well.